addon/background.js

```diff
@@ -49,3 +49,42 @@ browser.tabs.onUpdated.addListener((tabId, changeInfo, tabInfo) => {
             });
     }
 });
+
+async function freeze_active_tab(tab) {
+    // TODO: Is this try-catch unnecessary since it's being called below in a try-catch?
+    try {
+        await browser.tabs.executeScript(
+            tab.id,
+            {file: '/contentScript.js'}
+        );
+        const html = await browser.tabs.sendMessage(
+            tab.id,
+            {
+                type: 'freeze',
+                options: {
+                    wait: 0,
+                    shouldScroll: false
+                }
+            }
+        );
+        await download(html, {saveAs: true});
+    } catch(e) {
+        console.error(e);
+    }
+}
+
+browser.commands.onCommand.addListener((command) => {
+    if (command === "freeze-page") {
```

Comment:
single quotes, just for consistency

Reply:
I feel like I'm making this oversight a lot, and it frustrates me. It isn't the sort of mistake I usually make. I will make a point to eliminate these in the future.

Reply:
Huh, I *do* have eslint set up. I'm surprised I don't have it flagging this.